src/conversions/uuid.rs

```diff
@@ -0,0 +1,119 @@
+#![cfg(feature = "uuid")]
+//! TODO
+use uuid::Uuid;
+
+use crate::exceptions::PyValueError;
+use crate::sync::GILOnceCell;
+use crate::types::any::PyAnyMethods;
+use crate::types::bytes::PyBytes;
+use crate::types::dict::IntoPyDict;
+use crate::types::{PyBytesMethods, PyType};
+use crate::{
+    intern, Bound, FromPyObject, IntoPy, Py, PyAny, PyObject, PyResult, Python, ToPyObject,
+};
+
+impl FromPyObject<'_> for Uuid {
+    fn extract_bound(ob: &Bound<'_, PyAny>) -> PyResult<Self> {
+        // Not propagating Err up here because the UUID class should be an invariant.
+        let uuid_cls = get_uuid_cls(ob.py()).expect("failed to load uuid.UUID");
+        let py_bytes: Bound<'_, PyBytes> = if ob.is_exact_instance(&uuid_cls) {
+            ob.getattr(intern!(ob.py(), "bytes"))?.downcast_into()?
```

Comment:
Pydantic uses the int value to convert to the rust uuid. So dump question: Which one will be faster, int, bytes or does it not matter.

ref pydantic/pydantic-core#1362

Reply:
Ahhh, that's a good point. I had assumed the `bytes` argument had the lowest overhead, but looking into the `UUID` class source the `bytes` argument is just a proxy to passing an integer anyways:

```python

        if bytes is not None:
            if len(bytes) != 16:
                raise ValueError('bytes is not a 16-char string')
            assert isinstance(bytes, bytes_), repr(bytes)
            int = int_.from_bytes(bytes)  # big endian
```

https://github.com/python/cpython/blob/main/Lib/uuid.py#L192

I'll go ahead and switch to that 👍